_posts/en/newsletters/2021-03-24-newsletter.md

```diff
@@ -0,0 +1,299 @@
+      in a fraction of the time---making any protection from P2PKH-style
+      hash indirection irrelevant.  A third respondent
+      [proposed][harding bounty] constructing special Bitcoin addresses
+      that could only be spent from by someone making progress on fast
+      QCs; users could voluntarily donate bitcoins to the addresses to
+      create an incentivized early warning system.
+
+    - *We could add a hash-style address after taproot is activated:* if
+      a significant number of users really do think the sudden
+      appearance of a powerful QC is a threat, we could use a follow-up
+      soft fork to [add][dashjr quantum] an alternative P2PKH-style taproot
+      address type that uses hashes.  However, this has consequences
+      that caused several respondents to oppose it:
+
+        - It'll create confusion
+
+        - It'll use more block space
+
+        - It [reduces][poelstra anon] the size of taproot's anonymity
+          set, both directly and when protocols such as [ring signature
+          membership proofs][nick ring] or [Provisions][] are being used
+
+    - *Bandwidth/storage costs versus CPU costs:* it's [possible][rubin
+      recovery] to eliminate the extra 32-byte storage overhead from
+      hash indirection by deriving the public key from a signature and
+      the transaction data it signs, a technique called *key recovery*.
+      Again, this was opposed.  Key recovery requires a [significant amount of CPU][corallo
+      recovery overhead] that would slow down nodes and also prevents
+      the use of schnorr batch validation that can make historic block
+      verification up to three times faster.  It also [makes][poelstra
+      slowdowns] anonymous membership proofs and related techniques both
+      harder to develop and much more CPU intensive.  There
+      [may][poelstra patent story] also be a patent issue.
+
+  As of this writing, it appears the mailing list discussion has
+  concluded without any obvious loss of community support for taproot.
+  As researchers and businesses continue improving the state of the art
+  in quantum computing, we expect to see future discussions about how to
+  best keep Bitcoin secure.
+
+- **Weekly taproot activation meetings:** ten weekly meetings to discuss
+  details related to activating [taproot][topic taproot] have been
+  scheduled for each Tuesday at 19:00 UTC in the
+  [##taproot-activation][] IRC channel, with the first
+  [meeting][activation meeting log] occurring yesterday (March 23rd).
+
+## Changes to services and client software
+
+*In this monthly feature, we highlight interesting updates to Bitcoin
+wallets and services.*
+
+FIXME:bitschmidty
+
+## Releases and release candidates
+
+*New releases and release candidates for popular Bitcoin infrastructure
+projects.  Please consider upgrading to new releases or helping to test
+release candidates.*
+
+- [C-Lightning 0.10.0-rc1][C-Lightning 0.10.0] FIXME:harding
+
```

Comment:
Perhaps add https://github.com/rust-bitcoin/rust-lightning/releases/tag/v0.0.13 (although there aren't any release notes :slightly_frowning_face: )

Reply:
Covered last week.  :-)  https://bitcoinops.org/en/newsletters/2021/03/17/#rust-lightning-0-0-13

Reply:
You have a time machine?!

![image](https://user-images.githubusercontent.com/1063656/112133432-9df69a80-8bc3-11eb-9861-e3b9570f6a0c.png)